config/prow/plugins.yaml

```diff
@@ -254,6 +255,7 @@ lgtm:
   - kubernetes-sigs/cluster-api
   - kubernetes-sigs/cluster-api-operator
   - kubernetes-sigs/cluster-api-provider-azure
+  - kubernetes-sigs/cluster-api-provider-vsphere
```

Comment:
This is super useful. It stores the git tree hash in a comment after an lgtm comment.

If afterwards the commits are squashed without a change to files the lgtm is preserved